spec/acceptance/foreman_basic_spec.rb

```diff
@@ -55,4 +55,24 @@ class { 'foreman':
 
     it_behaves_like 'the foreman application', { expected_login_url_path: '/users/extlogin' }
   end
+
+  # needs to happen after GSSAPI, something is wrong with its cleanup
+  context 'in a Container' do
+    before(:context) { purge_foreman }
+    describe 'in a Container' do
+      it_behaves_like 'an idempotent resource' do
+        let(:manifest) do
+          <<~PUPPET
+          class { 'foreman':
+            deployment_mode => 'container',
+            db_host => 'localhost',
+            db_manage_rake => false,
```

Comment:
I wonder if these two could be made more "dynamic"?

Reply:
What do you mean? That `db_manage_rake` would be `undef` by default and the deployment mode determines the value?

Reply:
yeah, same for host (there is not really a socket in the container)